templates/group_vars_mash_servers

```diff
@@ -1596,6 +1601,55 @@ wetty_container_labels_traefik_tls_certResolver: "{{ devture_traefik_certResolve
 
 
 
+# role-specific:cadvisor
+########################################################################
+#                                                                      #
+# cadvisor                                                             #
+#                                                                      #
+########################################################################
+
+cadvisor_enabled: false
+
+cadvisor_identifier: "{{ mash_playbook_service_identifier_prefix }}cadvisor"
+
+cadvisor_uid: "{{ mash_playbook_uid }}"
+cadvisor_gid: "{{ mash_playbook_gid }}"
+
+# Only enable Traefik labels if a hostname is set (indicating that this will be exposed publicly)
```

Comment:
```suggestion
cadvisor_uid: "{{ mash_playbook_uid }}"
cadvisor_gid: "{{ mash_playbook_gid }}"

cadvisor_base_path: "{{ mash_playbook_base_path }}/{{ mash_playbook_service_base_directory_name_prefix }}cadvisor"

# Only enable Traefik labels if a hostname is set (indicating that this will be exposed publicly)
```

Without this override,  cAdvisor uses the role defaults and gets installed to `/mash-cadvisor`